CHANGES.rst

```diff
@@ -2,6 +2,34 @@
  Release history
 =================
 
+Version 7.0.2
+=============
+
+Released: 2023-04-03
+
+This is a bugfix release that fixes some test interaction issues and
+automates the documentation build.
+
+Fixes
+-----
+* Fix test-ordering issues with egg-based tests that modify ``pkg_resources``
+  global state. (#564)
+
+Documentation
+-------------
+* Reorganise the gh-pages documentation. The main gh-pages page
+  (https://docs.enthought.com/envisage) now matches the contents of the GitHub
+  ``main`` branch, while subdirectories (e.g.,
+  https://docs.enthought.com/envisage/7.0) provide documentation for released
```

Comment:
Note for reviewers: this URL is currently a 404. If the documentation build workflow works as intended, it'll become live shortly after the release is made.